programming/sr/cheat_sheet.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: page
+title: API Quick Reference
+---
+
+# SR API Quick Reference
+
+This page contains a quick guide to the `sr.robot3` API.
+
+For more information, make sure you check the rest of the documentation.
+
+## Initialising your robot
+
+### Standard Initialisation
+
+~~~~~ python
+R = Robot()
+~~~~~
+
+### Initialisation with manual start button
+
+~~~~~ python
+R = Robot(auto_start=True)
+
+# Code here runs before the start button is pressed
+
+R.wait_start()
+~~~~~
+
+### Initialisation with extra logging
+
+You can also tell the robot to print extra logging information, although this is quite noisy.
+
+~~~~~ python
+R = Robot(verbose=True)
+~~~~~
+
+## Selecting which board to control
+
+If you only have one board of a given type plugged into your robot, then you can use it's singular name:
```

Comment:
grammar nit: "its" is possessive, "it's" is a contraction of "it is". The trick I use to remember this is that "his" and "hers" also don't have apostrophes.

Reply:
*typo*, thanks for spotting

Reply:
0558becfaa613c15c4e13b4a4906753864e975ae